### DISCUSSION TITLE
Information for the CalcAUTOCORR2D and CalcAUTOCORR3D methods

### DISCUSSION THREAD

**Discussion by chrisdid4** [rdkit/rdkit#5098] [Q&A] [answered]
Hello everyone. To the best of my knowledge these methods  (CalcAUTOCORR2D and  CalcAUTOCORR3D) return a vector with 2d and 3d descriptors respectively. I would like to know each individual descriptor which is calculated using these methods. 
(For example something like this : the first value of the returned vector from the CalcAUTOCORR2D method is the molecular weight of the molecule etc.)

Thank you in advance,
Best,
Christos

**Comment by greglandrum** [comment, marked as answer]
Hi @chrisdid4, the AUTOCORR2D and AUTOCORR3D descriptors implemented in the RDKit are described in this RDKit UGM presentation from @thegodone : https://github.com/rdkit/UGM_2017/blob/master/Presentations/Godin_3D_Descriptors.pdf
